src/libs/brand-utils.test.js

```diff
@@ -1,7 +1,8 @@
-import { isBrand } from 'src/libs/brand-utils'
+import { getEnabledBrand, isBrand } from 'src/libs/brand-utils'
+import { brands, defaultBrand } from 'src/libs/brands'
 
 
-describe('isBrand', () => {
+describe('brand-utils', () => {
```

Comment:
`describe` blocks can be nested or there can be multiple `describe` blocks at the top level. This test could have one describe block per function.

https://jestjs.io/docs/api#describename-fn

Reply:
Done in [7f0d09b](https://github.com/DataBiosphere/terra-ui/pull/3625/commits/7f0d09b01e800bd7b193aff2762af41beb7ced07).